### DISCUSSION TITLE
Details pop-up not visible when window is narrower than 1236px

### DISCUSSION THREAD

**Discussion by wezm** [community/community#10677] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm not sure what it's real name is but in the search results you can click on a symbol and get a box in the top right that looks like this:

<img width="398" alt="Screenshot from 2022-01-28 17-01-39" src="https://user-images.githubusercontent.com/21787/151502285-7cdfbf61-bf8b-4d2d-bdf6-65429356b1c8.png">

Recently I would click on it and the symbol would highlight but nothing else would happen. I discovered that if I mad my window slightly wider then it would appear. Specifally the break point seems to be 1236 pixels: it's visible with windows that wide or wider but hidden on 1235.

Browser: Firefox 96.0 (on Linux)

**Comment by colinwm** [comment, marked as answer]
That's intentional behaviour, since in lower screen widths it tends to interfere with/overlap the code.

Any suggestions on how you'd want it to behave?

**Reply by wezm** [reply]
I think that 1235 is still plenty wide enough to show it. I think it it doesn't really interfere with the code because you can scroll the code under it and close it if needed. It's a user initiated action with a close button so I think it could be shown on almost all widths bigger than mobile.
